### PR TITLE
Remove legacy testsuite

### PR DESCRIPTION
**Pull request by dgdavid**
## Problem

> The old legacy testsuite is a blocker for the evolution of YaST and is causing breakage in Travis. — @ancorgs in https://github.com/yast/yast-squid/pull/33

## Solution

In most of YaST modules, we managed to migrate it to RSpec. But in other cases, such as in this module, the solution has been to completely **drop it**. 

More details in https://github.com/yast/yast-users/pull/230 and https://trello.com/c/lmE34oq8/1021-3-drop-old-testsuite.

## Other changes

Additionally, this PR removes the obsolete _pluglib-bindings_ too, as proposed in #22. @mvidner [explained it](#pullrequestreview-419393706), 

> they were used for LiMaL which was dropped after SLE 12, and also for old yast2-storage

With regard to re-enabling the Perl syntax check, it is not possible anymore. Again, @mvidner [wrote down a great explanation](#issuecomment-634800168),

> The syntax check implies parsing all `use`d packages, including `YaST::YCP`. Using `-I src` and symlinking `src/YaST` to `.` would help in finding `YCP.pm` BUT that one loads its binary component (compiled from `YCP.cc`) and we simply do not have it at the time the syntax check is called.

### CODE REVIEW

File: package/yast2-perl-bindings.spec [hunk 1]
```diff
@@ -74,8 +74,6 @@ rm $RPM_BUILD_ROOT/%{perl_vendorarch}/auto/YaST/YCP/libYCP.la
 %{perl_vendorarch}/auto/YaST/YCP/libYCP.so*
 %dir %{yast_moduledir}
 %{yast_moduledir}/YaPI.pm
-%{yast_ydatadir}/devtools/pluglib-bindings
```
Comment (mvidner) [inline package/yast2-perl-bindings.spec:77]: pluglib-bindings are NOT the legacy testsuite

They can be probably dropped too, but they're a very different thing. They were used for LiMaL which was dropped after SLE 12.

Comment (mvidner) [reply package/yast2-perl-bindings.spec:77]: OK, that was already proposed in #22. Meanwhile I have re-checked that nothing uses pluglib-bindings. Just mention it in the changelog and the PR description.

**Comment by mvidner** [comment]
About the failing Perl syntax check: it CANNOT be re-enabled.

When the checks were first put in place, it was failing in pluglib-bindings which we are going to drop, but that was just a coincidence.

The syntax check implies parsing all `use`d packages, including `YaST::YCP`. Using `-I src` and symlinking `src/YaST` to `.` would help in finding `YCP.pm` BUT that one loads its binary component (compiled from `YCP.cc`) and we simply do not have it at the time the syntax check is called.

File: package/yast2-perl-bindings.spec (hunk 1 above)
Comment (dgdavid) [reply package/yast2-perl-bindings.spec:77]: > Just mention it in the changelog and the PR description.

I'll do, thanks!

**Comment by dgdavid** [comment]
Thanks a lot for the explanation @mvidner :)

File: package/yast2-perl-bindings.changes [hunk 2]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed May 27 14:52:31 UTC 2020 - David Diaz <dgonzalez@suse.com>
+
+- Dropped legacy testsuite (related to bsc#1138668).
```
Comment (ancorgs) [inline package/yast2-perl-bindings.changes:4]: Dropping of pluglib-bindings should be mentioned here

**Pull request merged**